BookOfHabits/BookOfHabits/Controllers/CoinsController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using AutoMapper;
+using BookOfHabits.Requests.Cains;
+using BookOfHabits.Responses.Coins;
+using BookOfHabitsMicroservice.Application.Models.Coins;
+using BookOfHabitsMicroservice.Application.Services.Abstractions;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BookOfHabits.Controllers
+{
+    [ApiController]
+    [Route("api/v1/[controller]")]
+    public class CoinsController(ICoinsApplicationService coinsApplicationService,
+                                 IChooseHabitApplicationService chooseHabitApplicationService,
+                                 IMapper mapper) : ControllerBase
+    {
+        [HttpGet("room/{roomId:guid}")]
```

Comment:
add routes for your methods and comments. For example [HttpPut("CreateCoins")]

Reply:
I prefer minimal routes. if there are enough controller names and it's unique, I'll leave it that way.